src/vs/workbench/contrib/terminal/browser/xterm/decorationAddon.ts

```diff
@@ -266,7 +267,8 @@ export class DecorationAddon extends Disposable implements ITerminalAddon {
 		if (!this._terminal || mark.hidden || mark.marker.isDisposed || !defaultColor) {
 			return undefined;
 		}
-		const decoration = this._terminal.registerDecoration({ marker: mark.marker, overviewRulerOptions: { position: 'left', color: defaultColor.toString() }, height: mark.height });
+		const height = mark.endMarker ? mark.endMarker.line - mark.marker.line + 1 : 1;
+		const decoration = this._terminal.registerDecoration({ marker: mark.marker, anchor: 'left', foregroundColor: Color.cyan.toString(), height });
```

Comment:
Can this use the same infrastructure as command navhere:

https://github.com/microsoft/vscode/blob/eb3debdd9abc78bd0b5e5487a6fff3e198f895df/src/vs/workbench/contrib/terminal/browser/xterm/commandNavigationAddon.ts#L178-L211

So `DecorationAddon` handles command and mark/point of interest decorations, whereas the `MarkNavigationAddon` handles showing the temporary highlights.

Reply:
sounds good and we're moving away from height in favor of multiple decorations - tried that with dummy values/markers yesterday and couldn't get them to show up for some reason